components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -0,0 +1,173 @@
+$white:#ffffff;
+$black:#000000;
+$light-gray:#aaaaaa;
+$dark-gray:#2e2e2e;
+$gold:#c28e0e;
+
+.intro {
+	background-color: #c28e0e;
+	padding: 1.5em;
+
+
+	&__title {	
+	font-size: 2em;
+	line-height: 1.3em;
+	color: $black;
+	text-align: center;
+	text-transform:uppercase;
+	padding: .5em 0;
+	}
+	
+	&__issue {
+	font-size: 1em;
+	line-height: 1em;
+	color: $black;
+	text-align: center;
+	padding: 0;
+	}
+	
+	&__issue::after {
+	content:" ";
+	display: block;
+	width: 30%;
+	height: 3px;
+	margin: .3em auto !important;
+	background-color: $white;	
+	}
+	
+	&__description {
+	font-size: .9em;
+	line-height: 1.4em;
+	color: $black;
+	text-align: center;
+	padding: 1.5em 0 0 0;
+	}
+}
+
+
+.section {
+	background-color: $dark-gray;
+	padding: 1.5em;
+	
+	&__heading {
+	font-family:Impact,"Helvetica Neue",Helvetica,sans-serif;
+	color: $white;
+	font-size: 1.6em;
+	line-height: 1.2em;
+	text-transform:uppercase;
+	padding: .5em 0 0 0;
+	}
+	
+	&__heading::after {
+	content:" ";
+	display: block;
+	width: 20%;
+	height: 3px;
+	margin: .2em 0 0 0 !important;
+	background-color: $gold;	
+	}
+}
+
+.story {
+	margin: 0;
+	padding: 0;
+		
+	&__title {
+	color: $gold;
+	font-size: 1.4em;
+	line-height: 1.3em;
+	text-align: left;
```

Comment:
do we need text-align here? (and a few other places)

Reply:
I would say that `text-align:left;` is only necessary if a future story block is going to have its text aligned to the center by default. If that becomes the case in the future, we can specify the `text-align:left;` then. Otherwise, we can remove it for now.